StyleCop.Analyzers/StyleCop.Analyzers.Test.CSharp11/Lightup/IImportScopeWrapperCSharp11UnitTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// Tests calls where nullability rules are not obeyed
+#pragma warning disable CS8604 // Possible null reference argument.
```

Comment:
Just use `!` at the point where you intentionally break the rules in the test

Reply:
Updated